util/test/checkChplInstall

```diff
@@ -25,16 +25,30 @@
 # To use something other than $HOME/.chpl, set the CHPL_CHECK_INSTALL_DIR
 # environment variable.
 
+function log_date()
```

Comment:
I don't really like printing out the timestamps with each message. `make check` shouldn't take more than a few seconds, so I don't really think it's useful information